students/Sean_Tasaki/Lesson3/strformat_lab.py

```diff
@@ -0,0 +1,55 @@
+"""
+Sean Tasaki
+5/10/2018
+Lesson03.strformat_lab
+"""
+def task_one():
+   
+    tuple1 = (2, 123.4567, 10000, 12345.67)
+    
+    results = 'file_{:0>4d} : {:3.2f} , {:.2e} , {:03.2e}'.format(*tuple1)
+    
+    print("Task One results:")
+    print(results)
+
+
+def task_two():
+    results = ('file_0002', 123.46, 1.00e+04, 1.23e+04)
+    print("Task Two results:")
+    print(f'{results[0]:s} :  {results[1]:.2f}, {results[2]:.2e}\, {results[3]:.2e}')
+
+
+def task_three(tuple):
+    list = []
+    for i in tuple:
```

Comment:
don't use `list` or `tuple` for var names, you are overwriting a python built-in